spec/controllers/application_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+describe ApplicationController, type: :controller do
+  describe '#authorize_request requires body to be json web encrypted' do
+    # test controller to test ApplicationController logic
+    controller(ApplicationController) do
+      def create
+        render json: decrypted_body, status: :ok
+      end
+    end
+
+    context 'and rejects them if not' do
```

Comment:
This seems like a strange style of naming contexts and specs to me. Contexts should start with 'when' and `it ` blocks should describe what they do: https://github.com/rubocop-hq/rspec-style-guide#context-descriptions

Reply:
i have added the gem `rubocop-rpec` to add linting rules to address your complaints